roles/horizon/tasks/main.yml

```diff
@@ -1,121 +1,92 @@
 ---
-- name: install apache
-  apt: pkg={{ item }}
-  with_items:
-    - apache2
-    - libapache2-mod-wsgi
-
 - name: lesscpy must be in apache PATH
   pip: name=lesscpy version=0.9j
 
 - name: get horizon source repo
-  git: repo={{ openstack.git_mirror }}/horizon.git
-       dest=/opt/stack/horizon
-       version={{ horizon.rev }}
-       update={{ openstack.git_update }}
+  git: |
```

Comment:
Shouldn't need the line continuation here. Just put the first key=value here and indent everything to that level below it.